examples/annotation_import/text.ipynb

```diff
@@ -39,6 +39,7 @@
         "* Classification radio \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # only supported with NDJson and for MAL imports 
</pre><p>Need to remove this comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1031/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>